app/jsx/components/AuthorListComp.jsx

```diff
@@ -52,20 +53,18 @@ class AuthorListComp extends React.Component {
           {editors}
           {advisors}
           {/* Note: the <a> more-link below cannot be a <Link>, else jquery dotdotdot can't recognize it */}
-
-          { <li><a href={item_link} className="c-authorlist__list-more-link">et al.</a></li> } 
+          {/* Kludge: only show et al if the total number of contributors is greater than 6 */}
+          { (total_contributors > 6) &&  <li><a href={item_link} className="c-authorlist__list-more-link">et al.</a></li> } 
```

Comment:
I did want to call out this one remaining instance of 'et al.' which matches the same logic as what's used in generating the standard citation (i.e. only use 'et al.' when there are more than 6 contributors). This seems safe, and using it only on the item page makes sense.